azure-pipelines.yml

```diff
@@ -51,6 +51,17 @@ jobs:
         - script: |
               env RUSTFLAGS="-Z sanitizer=leak" cargo test --target x86_64-unknown-linux-gnu
           displayName: cargo -Z sanitizer=leak test
+    - job: loom
+      displayName: "Loom tests"
+      pool:
+        vmImage: ubuntu-latest
+      steps:
+        - template: install-rust.yml@templates
+          parameters:
+            rust: nightly
+        - script: |
+              env RUSTFLAGS="--cfg loom" cargo test --test loom
```

Comment:
I wonder if this'll have to set `LOOM_MAX_PREEMPTIONS` as well or the tests will run _forever_. See https://github.com/tokio-rs/loom#handling-large-models

Reply:
Ah, interesting. I assume this is future proofing the CI job? Considering that it currently runs on my laptop "instantaneously"

Reply:
Either way, added in 04a8c15fdf51e136815dbb146ea7f1b95d51cd41!